### ISSUE TITLE
Value of last transaction should be prefixed with + if positive

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pkern/kitcard-reader#6]
```
What steps will reproduce the problem?
1. Read a card whose last transaction is a deposit.

What is the expected output? What do you see instead?
The app should show the value of the last transaction prefixed with a plus (+). 
Negative last transactions are already prefixed, so the other direction should 
also be specified.
```

Original issue reported on code.google.com by `philipp.kern` on 13 Sep 2012 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3593672f4241 for English and German.
```

Original comment by `philipp.kern` on 17 Sep 2012 at 3:08
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `philipp.kern` on 7 Oct 2012 at 8:20
- Changed state: **Verified**
